source/OptimaJet.DWKit.StarterApplication/Forms/Groups/UpdateForm.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Linq;
+using JsonApiDotNetCore.Internal;
+using Microsoft.EntityFrameworkCore;
+using Optimajet.DWKit.StarterApplication.Models;
+using OptimaJet.DWKit.StarterApplication.Repositories;
+using OptimaJet.DWKit.StarterApplication.Services;
+
+namespace OptimaJet.DWKit.StarterApplication.Forms.Groups
+{
+    public class UpdateForm : BaseForm
+    {
+        public UserRepository UserRepository { get; set; }
+        public GroupRepository GroupRepository { get; set; }
+        public ICurrentUserContext CurrentUserContext { get; }
+        public UpdateForm(
+            UserRepository userRepository,
+            GroupRepository groupRepository,
+            ICurrentUserContext currentUserContext)
+        {
+            UserRepository = userRepository;
+            GroupRepository = groupRepository;
+            CurrentUserContext = currentUserContext;
+         }
+        public bool IsValid(int id, Group group)
+        {
+            var retVal = true;
+            //If changing organization, validate the change
+            if (group.OwnerId > 0)
```

Comment:
the comment and the code don't say the same thing?

this is just checking that the group _has_ an owner? maybe? I guess in what situation would the ownerId `== 0`?

Reply:
Changed all of the zeros everywhere to VALUE_NOT_SET (which is a static set to 0).  Changed this if to != the same as the others